user-guide/Standard_Apps/Monitoring_Solutions/DITT/Installing_DITT.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Installing_DITT
+---
+
+# Installing DataMiner IT Tools
+
+## Prerequisites
+
+- DataMiner version 10.3.6 or higher
+
+- A DataMiner System [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud)
+
+## Deploying DITT
+
+1. Look up the [*DataMiner IT Tools* package](https://catalog.dataminer.services/details/package/5959) in the DataMiner Catalog.
+
+1. Click *Deploy*.
+
+1. Select the target DataMiner System, and confirm the deployment. The package will be pushed to the DataMiner System.
+
+   If your DataMiner System does not contain a DITT element yet, a new DITT element will automatically be created when the package is installed.
+
+> [!NOTE]
+>
+> - To update the Planned Maintenance tool, redeploy the [*DataMiner IT Tools* package](https://catalog.dataminer.services/details/package/5959). The tool will be updated, but no new DITT element will be created (as one should already exist in your DMS).
```

Comment:
@EstebanSkyline , why do you mention the PLM tool here? This seems like an error.

Reply:
Indeed, sorry about that.
Should be:   To update the DataMiner IT Tool, redeploy the....